bigquery_etl/glam/cli.py

```diff
@@ -0,0 +1,143 @@
+import click
+from google.cloud import bigquery
+from .utils import run
+from pathlib import Path
+import os
+
+ROOT = Path(__file__).parent.parent.parent
+assert (ROOT / "sql").exists(), f"{ROOT} is not the project root"
+
+
+@click.group()
+def glam():
+    """Tools for GLAM ETL."""
+    pass
+
+
+@glam.group()
+def glean():
+    """Tools for Glean in GLAM."""
+    pass
+
+
+@glean.command()
+@click.option("--project", default="glam-fenix-dev")
+@click.option("--dataset", default="glam_etl_dev")
+def list_daily(project, dataset):
+    """List the start and end dates for clients daily tables."""
+    client = bigquery.Client()
+    app_df = client.query(
+        f"""
+        WITH
+        extracted AS (
+            SELECT
+                DISTINCT REGEXP_EXTRACT(table_name, "(.*)__") AS app_id,
+            FROM
+                `{project}`.{dataset}.INFORMATION_SCHEMA.TABLES
+            WHERE
+                table_name LIKE "%clients_daily%" )
+        SELECT
+            app_id,
+            (app_id LIKE "%glam%") AS is_logical
+        FROM
+            extracted
+        ORDER BY
+            is_logical,
+            app_id
+    """
+    ).to_dataframe()
```

Comment:
This requires `pandas` as a dependency. However, printing a dataframe doesn't require any extra formatting.